diff -ur log4cpp_origin/include/log4cpp/Appender.hh log4cpp/include/log4cpp/Appender.hh
--- log4cpp_origin/include/log4cpp/Appender.hh	2017-07-13 07:39:11.000000000 +0200
+++ log4cpp/include/log4cpp/Appender.hh	2018-10-02 12:25:48.864629277 +0200
@@ -161,7 +161,6 @@
         static AppenderMapStorage &_appenderMapStorageInstance;
     };
 
-	static Appender::AppenderMapStorageInitializer appenderMapStorageInitializer; // static initializer for every translation unit
     typedef std::set<Appender *> AppenderSet;
 
 }
diff -ur log4cpp_origin/src/Appender.cpp log4cpp/src/Appender.cpp
--- log4cpp_origin/src/Appender.cpp	2017-07-13 07:39:11.000000000 +0200
+++ log4cpp/src/Appender.cpp	2018-10-02 12:22:14.930502350 +0200
@@ -12,6 +12,7 @@
 #include <iostream>
 
 namespace log4cpp {
+	static Appender::AppenderMapStorageInitializer appenderMapStorageInitializer; // static initializer for every translation unit
 	static int appenders_nifty_counter; // zero initialized at load time
 	static char appenderMapStorage_buf[sizeof(Appender::AppenderMapStorage)]; // memory for the nifty-counter singleton object
 	Appender::AppenderMapStorage &Appender::_appenderMapStorageInstance = reinterpret_cast<Appender::AppenderMapStorage&> (appenderMapStorage_buf); // memory for placement new
