diff --git CMakeLists.txt CMakeLists.txt
index 36d486544..56c6142e0 100644
--- CMakeLists.txt
+++ CMakeLists.txt
@@ -533,8 +533,6 @@ install(TARGETS sdrangelbench DESTINATION bin)
 
 #install files and directories
 if (CMAKE_SYSTEM_NAME EQUAL "Linux")
-install(DIRECTORY udev-rules DESTINATION share/sdrangel)
-install(FILES udev-rules/install.sh DESTINATION share/sdrangel/udev-rules PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/sdrbase.rcc DESTINATION bin)
 install(FILES desktop/sdrangel.desktop DESTINATION share/applications)
 install(FILES desktop/sdrangel_icon.png DESTINATION share/pixmaps)
